core/src/components/textarea/textarea.ionic.outline.scss

```diff
@@ -0,0 +1,72 @@
+@use "../../themes/ionic/ionic.globals.scss" as globals;
+@import "./textarea.vars";
```

Comment:
This is not used I think and it shouldn't be imported here in my opinion, as its a partial that holds a value that doesn't come from a token.

Reply:
https://github.com/ionic-team/ionic-framework/pull/29776/commits/ea98cf34619813eebbd090272c124cdb515ba27b